src/plan_bot.py

```diff
@@ -235,26 +271,21 @@ def process_post(
         post_record_update["reply_type"] = (
             "plan_cluster" if plan.get("is_cluster") else "plan"
         )
-    elif operation == "all_the_plans":
-        print("all the plans requested: ", post.id)
-
-        reply_string = build_all_plans_response_text(plans, post)
-        post_record_update["reply_type"] = "operation"
-        post_record_update["operation"] = "all_the_plans"
-    elif operation == "help":
-        print("help requested: ", post.id)
+    elif operation and operation in operations_map:
+        print(operation, "requested: ", post.id)
 
-        reply_string = build_help_response_text()
+        response_fn = operations_map[operation]['response']
+        reply_string = response_fn(plans, post)
         post_record_update["reply_type"] = "operation"
-        post_record_update["operation"] = "help"
+        post_record_update["operation"] = operation
     else:
         print("topic mismatch: ", plan_id, post.id, plan_confidence)
 
         reply_string = build_no_match_response_text(potential_matches, post)
         post_record_update["reply_type"] = "no_match"
 
     try:
-        did_reply = reply(post, reply_string, send=send, simulate=simulate)
```

Comment:
Nit-pick: arg order. Can you make send and simulate the last two args as in the function signature? They're kwargs so it won't matter, but they are kind of conceptually the last things